### DISCUSSION TITLE
Typescript Error: Cannot find type definition file for @testing-library/jest-dom .

### DISCUSSION THREAD

**Discussion by arnediar** [gpbl/react-day-picker#2628] [Support]
Hi. 

I've installed the "react-day-picker": "^9.4.0" in our React App, and typescript gives me an error warning in the node_modules/react-day-picker/tsconfig.json:

`Cannot find type definition file for '@testing-library/jest-dom'.
  The file is in the program because:
    Entry point of type library '@testing-library/jest-dom' specified in compilerOptionsts
Cannot find type definition file for '@types/jest'.
  The file is in the program because:
    Entry point of type library '@types/jest' specified in compilerOptionsts
Cannot find type definition file for 'jest'.
  The file is in the program because:
    Entry point of type library 'jest' specified in compilerOptionsts`
    
 Do I have to install these packages to be able to use react-day-picker?

**Comment by princeverma-ai** [comment]
did you got the solution , I'm having the same issue .

**Comment by princeverma-ai** [comment]
did you got the solution , I'm having the same issue .

**Comment by khanshifaul** [comment]
Installing these module solved the issue

`npm install --save-dev jest @types/jest @testing-library/jest-dom`

**Comment by gpbl** [comment]
Note - this issue is already being tracked in https://github.com/gpbl/react-day-picker/issues/2674.
